.travis.yml

```diff
@@ -91,3 +93,29 @@ notifications:
 cache:
   directories:
     - $HOME/docker
+
+before_deploy:
+  - mkdir -p deploy/$TRAVIS_COMMIT
+  - >
+      if [ "$TRAVIS_OS_NAME" == "osx" ]; then
```

Comment:
Nit: single square brackets should use `=`. Otherwise use `[[` to make it clear bash is required.